### PR TITLE
fix: add 'image-name' argument to correct the invalid image namespace(mix with image-name)

### PR DESCRIPTION
**Pull request by zyy17**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

fix: add 'image-name' argument to correct the invalid image namespace(mix with image-name)

Please explain IN DETAIL what the changes are in this PR and why they are needed:

- Summarize your change (**mandatory**)
- How does this PR work? Need a brief introduction for the changed logic (optional)
- Describe clearly one logical change and avoid lazy messages (optional)
- Describe any limitations of the current code (optional)

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

**Comment by zyy17** [comment]
Yet another mistake... I hope it will work after merging... :<

File: .github/workflows/dev-build.yml [hunk 1]
```diff
@@ -263,7 +264,8 @@ jobs:
         uses: ./.github/actions/build-images
         with:
           image-registry: ${{ vars.ACR_IMAGE_REGISTRY }}
-          image-namespace: ${{ env.IMAGE_NAMESPACE }}
+          image-namespace: ${{ vars.IMAGE_NAMESPACE }}
```
Comment (daviderli614) [inline .github/workflows/dev-build.yml:267]: question: use `${{ env.IMAGE_NAMESPACE }}` ?

**Review by shuiyisong — APPROVED** [review]
please resolve comment and let's go

File: .github/workflows/dev-build.yml [hunk 2]
```diff
@@ -263,7 +264,8 @@ jobs:
         uses: ./.github/actions/build-images
         with:
           image-registry: ${{ vars.ACR_IMAGE_REGISTRY }}
```
[the remark this answers is not in the document]
Comment (zyy17) [reply .github/workflows/dev-build.yml:266]: It's already defined.

**Pull request merged**